db/fdb_access.c

```diff
@@ -56,6 +59,9 @@ struct fdb_access {
     listc_t lst;
 };
 
+
+static hash_t *fdb_dbname_hash;
```

Comment:
this is not mutex protected; we have traps adding/removing from it, and sql access this concurrently

Reply:
Adding mutex. Initially I did not think this is going to be used that much anyway, but of course you are right : )